### ISSUE TITLE
Nailing Games Night

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#26]
## TL;DR
I'll organise an online games night, on the proposed date of October 13th. Explanation on how it will work is below and I'd love your feedback. If you're interested, [add yourself to this event](https://calendar.google.com/event?action=TEMPLATE&tmeid=MHFycDY3aDZtbTR0bjk1aWdoYjE2aWticnEgam9lQHBvc3Rob2cuY29t&tmsrc=joe%40posthog.com) so I have an idea of numbers. I'll add more detail to that event in the coming weeks. 

## Context
One of the ideas from the offsite was to enable us (we, the PostHoggerers) to get together for more social occasions - even if that's virtually. It should be something fun so that people want to do it, but organised in such a way that everyone can contribute. It should be something people want to do, but also completely optional so nobody misses out. It should be awesome, but also probably somewhat awkward and nerdy.

Solution: Nail Games. 

### Short term vision
I'll organise a first games on [the 13th of October](https://calendar.google.com/event?action=TEMPLATE&tmeid=MHFycDY3aDZtbTR0bjk1aWdoYjE2aWticnEgam9lQHBvc3Rob2cuY29t&tmsrc=joe%40posthog.com), then take feedback to see if we need to tweak the idea or if people completely hate it and want to never speak of it ever again. It'll be completely optional and last an hour or so. 

If the feedback is good, I can organise more and find other, new games to play in the future. I have lots of suggestions on that because games are cool.

### What will we play and how will it work?
We'll play games from one of the Jackbox Party Packs. These are collections of medium-length party games which are suitable for players of _any_ ability or skill level. There are about 20 games to choose from and they vary wildly - some are about drawing or making rude jokes, while others are telling convincing lies or trivia. 

Some of the games we might play could include [Dictionarium](https://www.jackboxgames.com/dictionarium/) (a game about inventing definitions for new words), [Quiplash](https://www.jackboxgames.com/quiplash/) (a game of writing rapid-fire puns) or [Trivia Murder Party](https://www.jackboxgames.com/trivia-murder-party-two/) (a Saw-themed pub quiz). 

All of the games are played using your phone as a controller, and you don't need a copy to play. All you need is a PC or Mac (to join a Zoom call and see the game) and your phone (to input your answers). Why not grab a beer while you're at it?

Most of the games support 8 players and begin with a tutorial. 

If we get significantly more than 8 players signed up then I'd love feedback on how people would like it to work: remaining players can spectate and take it in turns, or we could organise multiple events?

**Comment by joethreepwood** [comment]
Oh, and if you've played Jackbox before then definitely let me know that too. We may have to split into groups and I'd also love to know what everyone's favourite games are.

**Comment by tiina303** [comment]
I ran a couple of games meetings for my team back at Twitter. We considered the jackbox games too, but it was just wildly painful to get it working properly, your milage may vary. I can facilitate an alternate concurrent session if we have too many people, but it wouldn't be Jackbox as I never got that to work 😅 Also available if you'd like to run a trial session beforehand.

Additional game ideas:
1. [overcooked 2](https://store.steampowered.com/app/728880/Overcooked_2/): requires folks to purchase the game on Steam 23$, but it's really fun, co-op, up to 4 can play at a time, kind of fun to watch folks play for the first few times, can do 4 play, 4 watch (useful for the same level & learn what works). Setup takes time so folks should do that beforehand & be friends with the person running it.
2. https://skribbl.io/ - multiplayer draw/guessing game, max 12 players, fast rounds, easy, anyone can join/jump at any time
3. https://netgames.io/games/ codenames, one night ultimate werewolf, Avalon, Spyfall, Love Letter - better if everyone knows the rules beforehand
4. Boardgamearena has lots of options too, I have a premium account, e.g. [Can't Stop](https://boardgamearena.com/gamepanel?game=cantstop) (2-4 players); [Kingdomino](https://boardgamearena.com/gamepanel?game=kingdomino). (2-4 players) [7 wonders](https://boardgamearena.com/gamepanel?game=sevenwonders) (3-7 players), [Incan Gold](https://boardgamearena.com/gamepanel?game=incangold) (3-8 players) sadly no Root. Setup wise everyone needs an account (free) and be friends with the person who's running it (needs to be premium to play with friends iirc), ideally done before the meeting so we don't waste time.

Additional organizing notes:
1. Some people prefer to know the rules ahead  (don't like unexpected stuff as much as others), so it could be helpful for them if the plan is published e.g. in the event description a day before.
2. It's probable some folks will arrive late / leave early. Know what games support that well & plan accordingly, e.g. sometimes we'd start with the drawing game if we knew someone was running a bit late or play that in the end, when people would be dropping off at random times.

**Comment by joethreepwood** [comment]
@tiina303 Thanks for the suggestions! I've run dozens of Jackbox nights before with little problem, so curious what issues you experienced? Will absolutely share some of the rules beforehand, but my experience has usually been that people enjoy just jumping in and not having to prepare so much. 

I did think of Overcooked (which I love), but figured four people was maybe too small and fixed a group. Boardgamearena is a great shout though and I play both 7Wonders and King Domino there on the regular. 

(I did also think of Neptune's Pride. But I've played that before with colleagues and it became an all-consuming-productivity-destroyer. Still, you may like it!)

**Comment by samwinslow** [comment]
+1 to open-ended sorts of games like https://skribbl.io/ and https://www.jackboxgames.com/dictionarium/ !

**Comment by neilkakkar** [comment]
Don't think I can add myself to your event, so +1ing here! I ran a few at Bloomberg before, and yes sounds like a lot of fun! :D

**Comment by lottiecoxon** [comment]
Same issue as Neil -I would also like to be involved this sounds super fun!

**Comment by EDsCODE** [comment]
Can we play pico park 😬

**Comment by joethreepwood** [comment]
Never tried it. How would it work?
